mangaki/mangaki/utils/mal.py

```diff
@@ -247,6 +310,10 @@ def insert_into_mangaki_database_from_mal(mal_entries: List[MALEntry]):
                 **extra_fields
             )
 
+            if (title and not first_work_matching
+                and title.upper() in list(map(str.upper, titles))):
+                first_work_matching = work
```

Comment:
Good question, I only replicated the former behavior of the MAL import which is to collect similar works returned from MAL Search. IMHO, I'd rather only add the relevant works and ignore the rest. What do you think, @jilljenn ?